tests/pipelines.py

```diff
@@ -1,8 +1,19 @@
 """
 Some pipelines used for testing
 """
+from scrapy import ItemPipeline
 
-class ZeroDivisionErrorPipeline(object):
+
+class LegacyZeroDivisionErrorPipeline(object):
```

Comment:
According to kmike:

> We also need to make sure pipelines/middleware/extensions which are not subclasses of these base classes still work.

This is my solution; to test the old way of creating an item pipeline